### PR TITLE
refactor(cleanup): remove force-fetch-alert-labels-from-metaphysics ff

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **TYPE**

This PR solves [FX-4671]

### Description

remove force-fetch-alert-labels-from-metaphysics ff

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FX-4671]: https://artsyproduct.atlassian.net/browse/FX-4671?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dimatretyak — CHANGES_REQUESTED** [review]

**Comment by gkartalis** [comment]
converted to Draft, will address later (switched focus to artwork lists for now)

File: src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx [hunk 1]
```diff
@@ -338,10 +320,8 @@ const SavedSearchAlertEditFormContainer: React.FC<SavedSearchAlertEditFormProps>
   let labels: LabelEntity[] | undefined
   let pills: FilterPill[] | undefined
 
-  if (shouldFetchLabelsFromMetaphysics) {
-    labels = (savedSearch?.labels as LabelEntity[]) ?? []
-    pills = convertLabelsToPills(labels)
-  }
+  labels = (savedSearch?.labels as LabelEntity[]) ?? []
+  pills = convertLabelsToPills(labels)
```
[the remark this answers is not in the document]
Comment (dimatretyak) [reply src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx:324]: As far as I understand, we want to parse pill labels **from aggregations**, and not get pill labels from Metaphysics. If yes, then in that case we should remove the `labels` and `pills` variables and not pass `initialPills` prop [here](https://github.com/artsy/force/blob/e7abbce7896ef50fa354ea3a939a36d89219af4d/src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx#L331)

File: src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx [hunk 2]
```diff
@@ -486,12 +462,7 @@ export const SavedSearchAlertEditFormQueryRenderer: React.FC<SavedSearchAlertEdi
           return null
         }
 
-        if (
-          shouldFetchLabelsFromMetaphysics &&
-          props?.artistsConnection &&
-          props.me &&
-          props.viewer
-        ) {
+        if (props?.artistsConnection && props.me && props.viewer) {
```
Comment (dimatretyak) [inline src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx:465]: If we will parse pill labels from aggregations, then we have to delete [this](https://github.com/artsy/force/blob/e7abbce7896ef50fa354ea3a939a36d89219af4d/src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx#L465-L477)

File: src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (dimatretyak) [reply src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx:324]: Addition to the previous message: https://github.com/artsy/force/pull/12097/files#r1138655901

**Review by dimatretyak — APPROVED** [review]

**Pull request merged**